htdocs/admin/inc/admin_functions.inc.php

```diff
@@ -1577,4 +1630,19 @@ function parseDownloadLink($arr) {
 		}
 		return false;
 	}
-?>
+
+    function getAdminText(string $key): string
```

Comment:
For future refactorings I would avoid adding more functions to the functions.inc.php. Can create services with tests instead.

Reply:
Done 6942a1f